### PR TITLE
Provide more context to the water mask source transition

### PR DESCRIPTION
**Pull request by hjkristenson**

### CODE REVIEW

**Comment by hjkristenson** [comment]
@forrestfwilliams I haven't updated the changelog or applied a label yet, as I wasn't sure where things were in the release process. It looks like @cirrusasf also has another PR open, so I thought I'd wait until everyone's back before working this in.

File: docs/guides/insar_product_guide.md [hunk 1]
```diff
@@ -52,7 +52,10 @@ There is an option to apply a **water mask**. This mask includes coastal waters
 
 A GeoTIFF of the water mask is always included with the InSAR product package, but when this option is selected, the conditional water mask will be applied along with coherence and intensity thresholds during the phase unwrapping process. Water masking is turned off by default. 
 
-The water mask is generated by the HyP3 team using data from [OpenStreetMaps](https://www.openstreetmap.org/about){target=_blank} and/or [ESA WorldCover](https://esa-worldcover.org/en/about/about){target=_blank} depending on location. Areas within Canada, Alaska, and Russia are primarily covered by ESA WorldCover data, while the rest of the world is covered by OpenStreetMaps data. Water masks were previously generated from the GSHHG dataset.
+The water mask is generated by the HyP3 team using data from [OpenStreetMaps](https://www.openstreetmap.org/about){target=_blank} and/or [ESA WorldCover](https://esa-worldcover.org/en/about/about){target=_blank} depending on location. Areas within Canada, Alaska, and Russia are primarily covered by ESA WorldCover data, while the rest of the world is covered by OpenStreetMaps data.
```
Comment (forrestfwilliams) [inline docs/guides/insar_product_guide.md:55]:
```suggestion
The water mask is generated by the HyP3 team using data from [OpenStreetMaps](https://www.openstreetmap.org/about){target=_blank} and/or [ESA WorldCover](https://esa-worldcover.org/en/about/about){target=_blank} depending on location. Areas within Canada, Alaska, and Russia are primarily covered by ESA WorldCover data, while the rest of the world is covered by OpenStreetMaps data.

The water mask is available for all longitudes, and from -85 to 85 degrees latitude. Thus, the water mask is not available for the centers of the North Pole and South Pole.
```

Comment (hjkristenson) [reply docs/guides/insar_product_guide.md]: Addressed in [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/423/commits/69eb243b1b4c2ca630adf8af3bfa13ec2ba30ab6)

File: docs/guides/burst_insar_product_guide.md [hunk 2]
```diff
@@ -137,7 +137,10 @@ There is an option to apply a **water mask**. This mask includes coastal waters
 
 A GeoTIFF of the water mask is always included with the InSAR product package, but when this option is selected, the conditional water mask will be applied along with coherence and intensity thresholds during the phase unwrapping process. Water masking is turned off by default. 
 
-The water mask is generated by the HyP3 team using data from [OpenStreetMaps](https://www.openstreetmap.org/about){target=_blank} and/or [ESA WorldCover](https://esa-worldcover.org/en/about/about){target=_blank} depending on location. Areas within Canada, Alaska, and Russia are primarily covered by ESA WorldCover data, while the rest of the world is covered by OpenStreetMaps data. Water masks were previously generated from the GSHHG dataset.
+The water mask is generated by the HyP3 team using data from [OpenStreetMaps](https://www.openstreetmap.org/about){target=_blank} and/or [ESA WorldCover](https://esa-worldcover.org/en/about/about){target=_blank} depending on location. Areas within Canada, Alaska, and Russia are primarily covered by ESA WorldCover data, while the rest of the world is covered by OpenStreetMaps data.
```
[the remark this answers is not in the document]
Comment (hjkristenson) [reply docs/guides/burst_insar_product_guide.md]: Addressed in [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/423/commits/69eb243b1b4c2ca630adf8af3bfa13ec2ba30ab6)

**Review by AndrewPlayer3 — APPROVED** [review]

**Pull request merged**